spec/tbd_spec.rb

```diff
@@ -20,11 +20,10 @@ def get_clean_env
 
 RSpec.describe TBD do
   it "it can run all the measure tests" do
-    measure_dir = File.join(__dir__, "../../lib/measures")
+    measure_dir = File.join(__dir__, "../lib/measures")
```

Comment:
@macumber: path mix-up  :[ ... To the best of my knowledge, this is has been like this for a while. Anyone running `bundle exec rake` got the (false) impression everything was OK. Apologies to anyone who may have been scratching their heads with this.